pkg/scheduler/cache/event_handlers.go

```diff
@@ -59,7 +59,11 @@ func (sc *SchedulerCache) getOrCreateJob(pi *schedulingapi.TaskInfo) *scheduling
 func (sc *SchedulerCache) addTask(pi *schedulingapi.TaskInfo) error {
 	if len(pi.NodeName) != 0 {
 		if _, found := sc.Nodes[pi.NodeName]; !found {
-			return fmt.Errorf("node <%s> does not exist", pi.NodeName)
+			if sc.nodeInformer.Informer().HasSynced() {
```

Comment:
AFAIK, we already sync all informers when starting scheduelr.

Reply:
The time sequence of pods and node resources cannot be guaranteed.

Reply:
the problem is happened in the starting scheduler period to sync informers data

Reply:
I deleted the judgment."if sc.nodeInformer.Informer().HasSynced" , and  created the nodeinfo structure directly.

Reply:
what's the impact if there's no node object in node info?

Reply:
if there's no node object in node info, the nodeinfo status will be "NotReady" util the imformer receives the node object 
 to update the status to "Ready". So it is no influence to the volcano schduling and just saves the task info in its nodeInfo.

Reply:
that's great!